Core/CMP API Specification.md

```diff
@@ -176,22 +176,113 @@ cmpDisplayStatus: String, // possible values: hidden, visible, disabled
 
 signalStatus : String, // possible values: not ready, ready
 
-supportedAPIs : Array of string, // list of supported APIs (section ids and prefix strings), e.g. used while loading. Example: ["2:tcfeuv2","6:uspv1"] 
+// List of supported APIs (section ids and prefix strings), e.g. used while loading.
```

Comment:
can we get rid of the "e.g ...."? it sounds like it is only used during loading and can be ignored afterwards

Reply:
I think that makes sense